spec/services/proofing/resolution/progessive_proofer_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'rails_helper'
```

Comment:
Could people who are reviewing this PR please focus on this file?  I'm concerned about writing tests that are both effective and thorough.